docs/source/intro/platform.md

```diff
@@ -1,99 +1,107 @@
-schema against a set of previously-observed operations. This uses the
-trace warehouse, operation registry, and (typically) the client
-registry. As an example, an operation that references a missing field or
-an operation that doesn't pass a required argument to a field would
-cause an incompatibility error. The compatibility check runs statically,
-taking advantage of the schema's type definitions, so it doesn't require
-a running server.
-
-### Safelisting
-
-Apollo provides an end-to-end mechanism for _safelisting_ known clients
-and queries, a recommended best practice that limits production use of a
-GraphQL API to specific pre-arranged operations. There are two parts
-here. First, the Apollo CLI extracts all the queries from a client
-codebase, computes the over-the-wire subset of the query (stripping out
-the part that references the client's local schema), and stores it in
-the operation registry. Separately, an Apollo Server plugin synchronizes
-the list of pre-registered operations to the server, which then rejects
-queries that aren't present in its local copy.
+The Apollo platform helps you [build](#build-your-graph-with-apollo-server), [query](#query-your-graph-with-apollo-client), and [manage](#manage-your-graph-with-apollo-graph-manager) a **data graph**: a unified data layer that enables applications to interact with data from any combination of connected data stores and external APIs.
+
+A data graph sits between application clients and back-end services, facilitating the flow of data between them:
+
+![diagram](../images/index-diagram.svg)
+
+An Apollo data graph uses **GraphQL** to define and enforce the structure of this data flow.
+
+
+## Build your graph with Apollo Server
+
+Your data graph needs a service that processes GraphQL operations from application clients. This service communicates with back-end data sources to fetch and modify data as needed. To build this service, you can use [**Apollo Server**](https://www.apollographql.com/docs/apollo-server/).
+
+Apollo Server is an extensible, [open-source](https://github.com/apollographql/apollo-server) JavaScript GraphQL server. With it, you can define:
+
+* A [**GraphQL schema**](https://www.apollographql.com/docs/apollo-server/schema/schema/) that specifies all of the types and fields available in your graph
+* A collection of [**resolvers**](https://www.apollographql.com/docs/apollo-server/data/resolvers/) that specify how to populate each field of your schema with data from your back-end data sources
+
+You can deploy Apollo Server to any hosted or serverless environment. It supports a variety of popular [Node.js middleware](https://www.apollographql.com/docs/apollo-server/integrations/middleware/).
+
+And if you want your data graph to use a [federated architecture](#federate-your-graph-with-apollo-federation), Apollo Server includes extension libraries that enable it to act as every component of that architecture.
```

Comment:
Can we get more specific about what a federated architecture is and why you would want one? It might be worth breaking that into its own small section on data graph architecture.